Storage/SQL/BrowserDB.swift

```diff
@@ -220,7 +219,9 @@ open class BrowserDB {
             }
             return success
         }) {
-            // Err getting a transaction
+            // Error getting a transaction
+            log.error("Unable to get a transaction: \(error.localizedDescription)")
+            SentryIntegration.shared.sendSync(message: error.localizedDescription, tag: "BrowserDB", severity: .error)
```

Comment:
For this specific error is there not anything else you would like to log? Or will that be in another PR?